docs/content/en/main/concepts/authorization.md

```diff
@@ -241,6 +240,42 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+### Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
+In this case access to the requested resource is constrained by the field `verbs.restrict` of the underlying permission claim.
+
+If the requested resource type is claimed by different API Exports in the same workspace,
+the intersection of all restrict verbs of all APIExports is being granted.
+
+Valid values for the `verbs.restrict` field follow the same rules as the `verbs` field in PolicyRules:
+
+- It is required.
+- `"*"` matches all verbs. If present, it must be the only entry.
+- The list must not be empty.
```

Comment:
I just realized, this is wrong. We decided that empty list corresponds to "no verbs are allowed".

Reply:
fixed